.github/workflows/tests.yml

```diff
@@ -15,10 +15,18 @@ jobs:
     strategy:
       matrix:
         node-version: ["12", "13", "14", "15"]
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Any reason to not also include macOS here?

Reply:
No idea.

Reply:
@1st1 or @jaclarke is there a reason that macOS should not be added?

Reply:
I think it should be

Reply:
added